common/lib/capa/capa/capa_problem.py

```diff
@@ -710,12 +710,14 @@ def _extract_html(self, problemtree):  # private
             hint = ''
             hintmode = None
             input_id = problemtree.get('id')
+            answervariable = None
```

Comment:
I know some of the code here does not follow this, but it would be nice if the variable name was `answer_variable`.